ups/ci_hsc_gen2.table

```diff
@@ -15,5 +15,6 @@ setupRequired(sconsUtils)
 setupRequired(utils)
 setupRequired(pipe_drivers)
 setupRequired(daf_butler)
+setupRequired(ci_hsc)
```

Comment:
Should this be `testdata_ci_hsc` ... or maybe it should be both `ci_hsc` and `testdata_ci_hsc`?

Reply:
Another case where I missed the name change, good catch